### PR TITLE
Spaces group views

### PR DESCRIPTION
**Pull request by bug0001**
unwanted spaces issue

### CODE REVIEW

**Comment by jsangmeister** [comment]
Please always fill out the correct meta information, see https://github.com/OpenSlides/OpenSlides/wiki/Guideline-for-Issues-and-Pull-Requests

File: client/src/app/site/users/components/user-list/user-list.component.html [hunk 1]
```diff
@@ -67,9 +67,9 @@ <h2>{{ 'Participants' | translate }}</h2>
         <div class="groupsCell">
             <div *ngIf="getUserGroups(user)?.length">
                 <os-icon-container icon="people" noWrap="true">
-                    <span *ngFor="let group of getUserGroups(user); let last = last">
+                    <span *ngFor="let group of getUserGroups(user); let last = last" class="remove-spaces-group">
                         {{ group.getTitle() | translate }}
-                        <span *ngIf="!last">,</span>
+                        <span *ngIf="!last">,&nbsp;</span>
                     </span>
                 </os-icon-container>
```
Comment (GabrielInTheWorld) [inline client/src/app/site/users/components/user-list/user-list.component.html:74]: That is good, but you're missing some other places described in #799.

**Comment by bug0001** [comment]
Please specify in which area of the application "In the list of submitters, there is a space after the username of a submitter" takes place.

File: client/src/app/site/users/components/user-list/user-list.component.html [hunk 2]
```diff
@@ -67,9 +67,9 @@ <h2>{{ 'Participants' | translate }}</h2>
         <div class="groupsCell">
             <div *ngIf="getUserGroups(user)?.length">
                 <os-icon-container icon="people" noWrap="true">
-                    <span *ngFor="let group of getUserGroups(user); let last = last">
+                    <span *ngFor="let group of getUserGroups(user); let last = last" class="remove-spaces-group">
```
Comment (GabrielInTheWorld) [inline client/src/app/site/users/components/user-list/user-list.component.html:70]: You have to adjust the class here to your new class-name.

**Review by GabrielInTheWorld — CHANGES_REQUESTED** [review]

**Review by GabrielInTheWorld — APPROVED** [review]
Rebase and ensure, that the tests run successfully.

**Pull request merged**